admin/data/sql/migrations/20190308122704.do.add-sce-table.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE [mtc_admin].[sce] (
```

Comment:
Wondering if we should change this name? Apparently, SCE hasn't been used since 2013.  Perhaps something like schoolTimezone?